### PR TITLE
Ignore reporting metrics from forks

### PR DESCRIPTION
**Pull request by dabonnie**
Signed-off-by: Devin Bonnie <dbbonnie@amazon.com>

### CODE REVIEW

**Review by jikawa-az — APPROVED** [review]

**Comment by thomas-moulard** [comment]
Please document the problem you're trying to fix.

**Review by thomas-moulard — CHANGES_REQUESTED** [review]
See comment

**Review by thomas-moulard — APPROVED** [review]
approving to avoid enforcing a second review.

lgtm as long as:
(1) all repos are updated
(2) the PR points to a draft PR from a fork completely green w/ this fork applied

**Pull request merged**
